pytorch_lightning/loggers/wandb.py

```diff
@@ -141,8 +143,7 @@ def log_hyperparams(self, params: Union[Dict[str, Any], Namespace]) -> None:
     @rank_zero_only
     def log_metrics(self, metrics: Dict[str, float], step: Optional[int] = None) -> None:
         assert rank_zero_only.rank == 0, 'experiment tried to log from global_rank != 0'
-
-        self.experiment.log({'global_step': step, **metrics} if step is not None else metrics)
+        self.experiment.log(metrics, step=step + self._step_offset if step is not None else None)
```

Comment:
```suggestion
        self.experiment.log(metrics, step=(step + self._step_offset) if step is not None else None)
```
for visual clarity :)